x-pack/legacy/plugins/apm/server/routes/error_occurrence_alert.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as t from 'io-ts';
+import { createRoute } from './create_route';
+import { createAlert } from '../lib/alerting/error_occurrence/create_alert';
+
+export const errorOccurrenceAlertRoute = createRoute(core => ({
```

Comment:
I've created a route for this, because we have to execute several concurrent and sequential requests, and the server is a more robust environment for that kind of dependencies.